src/navigation/menu_bar.py

```diff
@@ -559,6 +560,9 @@ def do_open_link(self) -> None:
             # Validate url (whether internal or external)
             # Then open in new tab
             webbrowser.open_new_tab(word)
+    
+    def do_show_shortcuts(self) -> None:
+        self.show_message("Shortcuts","- `CTRL+K`: Open Top Tool Bar.\n- ESC: Focus cursor on the text field\n- CTRL+N: Start a new file\n- CTRL+S: Save current file\n- CTRL+O: Open an existing note\n- CTRL+Q: Exit the application\n- CTRL+A: Select Everything\n- CTRL+Z: Undo\n- CTRL+E: Turn text like :smile: into emoji\n- ALT+O: Open link under cursor")
```

Comment:
Not sure why the first shortcut is wrapped in backticks. I also think it's more conventional to put spacing around pluses, e.g., `CTRL + N` instead of `CTRL+N`. Aside from that, looks good.